docgen/src/conditional-display.md

```diff
@@ -31,6 +40,16 @@ const Content = createConnector({
 
 ## Conditionally displaying content based on hits results 
 
+Another example where conditional display may apply is: 
+
+**What if you want to display a particular content when the search made by the user don't return any results?**
```

Comment:
Same, replace the title with "Displaying content when there's no results" and remove introduction sentences